src/core/main.c

```diff
@@ -1959,8 +1963,9 @@ static int load_configuration(int argc, char **argv, const char **ret_error_mess
                 return r;
         }
 
-        /* Log level should be put in effect as soon as possible. */
+        /* Log level/target should be put in effect as soon as possible. */
         log_set_max_level(arg_default_log_max_level);
+        log_set_target(arg_default_log_target);
```

Comment:
nah, we can't do this. the log target is set differently depending on the environment we run in (i.e. --system on host, --system in container, or --user). see the code at the top of main.c that calls log_set_target() in various cases... We need to continue to honour that.